servers/src/main/java/tachyon/master/rawtable/RawTableMasterServiceHandler.java

```diff
@@ -21,13 +21,16 @@
 import tachyon.Constants;
 import tachyon.TachyonURI;
 import tachyon.exception.TachyonException;
+import tachyon.replay.ReplayCache;
 import tachyon.thrift.RawTableInfo;
 import tachyon.thrift.RawTableMasterService;
+import tachyon.thrift.RpcOptions;
 import tachyon.thrift.TachyonTException;
 import tachyon.thrift.ThriftIOException;
 
 public class RawTableMasterServiceHandler implements RawTableMasterService.Iface {
   private final RawTableMaster mRawTableMaster;
+  private final ReplayCache<Object> mReplayCacheLong = ReplayCache.newInstance();
```

Comment:
should no longer be called "mReplayCacheLong". Maybe also add a comment, and mention why this is of type "Object"